.travis.yml

```
@@ -4,15 +4,11 @@ cache:
     - $HOME/.cache/pip
 matrix:
   include:
-    - python: 3.4
+    - python: 3.7
       env: TOXENV=docs
-    - python: 3.4
-      env: TOXENV=py34
-    - python: 3.5
-      env: TOXENV=py35
     - python: 3.6
       env: TOXENV=py36
 install:
-  - pip install tox
+  - sudo apt install make
```

development dependencies are currently exposed by https://github.com/shaarli/python-shaarli-client/blob/master/requirements/dev.txt

this will fetch too many unneeded dependencies though (in a CI context), we might want to introduce a new `requirements/ci.txt` file that pins CI dependencies and include it in `requirements/dev.txt`:

```suggestion
  - pip install -r requirements/ci.txt
```